### ISSUE TITLE
RequestError

### ISSUE THREAD

**Issue by dutzi** [delianides/kap-vercel#6]
**macOS version:**    Monterey (12)
**Kap version:**      3.4.1

```
RequestError

RequestError: certificate has expired
    at ClientRequest.<anonymous> (/Applications/Kap.app/Contents/Resources/app.asar/node_modules/got/source/request-as-event-emitter.js:178:14)
    at ClientRequest.origin.emit (/Applications/Kap.app/Contents/Resources/app.asar/node_modules/@szmarczak/http-timer/source/index.js:37:11)
```

**Comment by Soben** [comment]
Receiving the same error, with Mac OS Version Big Sur (11.6) and the Kap 3.4.2

Also relates to #5

**Comment by delianides** [comment]
Resolved with 0.4.4
